trypybindings/output.sas

```diff
@@ -0,0 +1,71 @@
+begin_version
```

Comment:
Having this file in the repo for the prototype is fine but of course in the long runs, we should remove it again.